gateway/ln-gateway/src/lib.rs

```diff
@@ -456,6 +453,8 @@ impl Drop for LnGateway {
 pub enum LnGatewayError {
     #[error("Federation client operation error: {0:?}")]
     ClientError(#[from] ClientError),
+    #[error("Lightning rpc operation error: {0:?}")]
+    LnrpcError(#[from] tonic::Status),
```

Comment:
just a nit: since `ln` and `rpc` are two different nouns, maybe call this `LnRpcError`

Reply:
💯